backends/tc/README.md

```diff
@@ -0,0 +1,32 @@
+# TC backend
```

Comment:
Is there any requirement on the kernel version?

Reply:
Updated the version info